### PR TITLE
CX-2445 Improvement/woopra safe window

### PR DESCRIPTION
**Pull request by rfreitas**
# Problem

Woopra is polluting Window, which can be a problem if the host app is using woopra as well, since both instance can conflict.

# Solution

Wrap woopra in a safe window object which is exclusively owned by woopra instance (who think it's a real window). This way woopra pollutes the fake/safe window only and not the real one,

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a] Have new automated tests been implemented?
- [n/a] Have new manual tests been written down?
- [x] Have tests passed locally?
- [n/a] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-502-pr-onfido-sdk-ui-onfido.surge.sh

File: src/Tracker/safeWoopra.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      },
+        set: value=>{ window[key] = value }
+      }
+    )
+  })
+}
+SafeWindow.prototype = window;
+
+
+const windowKey = "onfidoSafeWindow8xmy484y87m239843m20";
```
Comment (josokinas) [inline src/Tracker/safeWoopra.js]: [Minor] As the hash is used multiple times, might be better to have it as a constant somewhere.

Comment (rfreitas) [reply src/Tracker/safeWoopra.js]: I had tried it with a runtime constant and it hadn't worked. However, I hadn't though of making it a literal constant. It worked actually! Commit has been pushed. thanks.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://502-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-502-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**